CodeEdit/Features/TerminalEmulator/Model/Shell.swift

```diff
@@ -11,19 +11,22 @@ import Foundation
 enum Shell: String, CaseIterable {
     case bash
     case zsh
+    case fish
 
     var url: String {
         switch self {
         case .bash:
             "/bin/bash"
         case .zsh:
             "/bin/zsh"
+        case .fish:
+            "/opt/homebrew/bin/fish"
```

Comment:
@nis-ship-it Is this safe to assume that the fish binary will always be installed by homebrew?

Reply:
Yes it might be different so I will try to figure on how to obtain path

Reply:
We might look at how other editors handle this